front/tests/003-study-management.spec.ts

```diff
@@ -9,6 +9,11 @@ import HomePage from './pages/home-page-model';
 import StudyPage from './pages/study-page-model';
 import { getProject, postApiRequest } from './utils/api-setup';
 
+test.use({
+  launchOptions: {
+    slowMo: 10,
+  },
```

Comment:
Why do you slow this test ? Add a comment to answer this question, so the next person who will read this code will know why you needed to do this